ThroughputTest/ReceiverTask.cs

```diff
@@ -139,84 +141,47 @@ async Task ReceiveTask(string path)
                 {
                     nsec = sw.ElapsedTicks;
                     // we're going to unblock the receives after 10 seconds if there's no pending message
-                    receiver.ReceiveAsync(Settings.ReceiveBatchCount, TimeSpan.FromSeconds(10)).ContinueWith(async (t) =>
+                    Task<IReadOnlyList<ServiceBusReceivedMessage>> t = receiver.ReceiveMessagesAsync(Settings.ReceiveBatchCount, TimeSpan.FromSeconds(10));
```

Comment:
Why was the ContinueWith removed?

Reply:
@JoshLove-msft I have updated, please review.